scheduler/plugin/plugin_test.go

```diff
@@ -1807,3 +1886,7 @@ func requireError(t *testing.T, err error, expectedError string) {
 		t.Fatalf("unexpected error: want %v, have %v", want, have)
 	}
 }
+
+func makeTestWaitingPod(pod *v1.Pod, plugins map[string]*time.Timer) framework.WaitingPod {
+	return util.NewWaitingPod(pod, plugins)
+}
```

Comment:
Do we need this function. It's just an alias.

Reply:
i will remove it